src/main/java/fi/okm/jod/yksilo/controller/profiili/ToimenkuvaController.java

```diff
@@ -58,7 +59,7 @@ ResponseEntity<Void> add(
             .path("/{toimenkuvaId}")
             .buildAndExpand(toimenkuvaId)
             .toUri();
-    return ResponseEntity.created(location).build();
+    return ResponseEntity.created(location).body(new IdDto<>(toimenkuvaId));
   }
 
   @PostMapping("/{toimenkuvaId}")
```

Comment:
I would reserve POST -method for endpoints that create entities, and use PATCH for these instead.

(PATCH in case of partial updates, which I guess all updates are)

Reply:
For some reason comment missed the target by a few lines, was related to this part:

```
  @PostMapping("/{id}")
  @ResponseStatus(HttpStatus.NO_CONTENT)
  void update(
```